torchci/pages/metrics.tsx

```diff
@@ -186,7 +171,7 @@ function TTSPanel({
         },
       ]}
       dataGridProps={{ getRowId: (el: any) => el.name }}
-      useClickHouse={useClickHouse}
+      useClickHouse={true}
```

Comment:
nit: you could prob just have this param default to true now, right?

Reply:
It got deleted in rebase